src/plugins/unified_histogram/public/layout/hooks/use_lens_suggestions.ts

```diff
@@ -122,7 +125,7 @@ export const useLensSuggestions = ({
   }, [currentSuggestion, dataView, query, timeRange, data, lensSuggestionsApi]);
 
   useEffect(() => {
-    const newSuggestionsDeps = getSuggestionDeps({ dataView, query, columns });
+    const newSuggestionsDeps = getSuggestionDeps({ dataView, query, columns, table });
```

Comment:
Hey, while testing I noticed that refreshing grid data resets the currently selected suggestion. This should not happen ideally, right?

![Jan-24-2024 11-06-40](https://github.com/elastic/kibana/assets/1415710/4ad6353c-a8a1-4c00-b10d-5860bbeb0e0e)


Btw, the bars chart has some weird issues with rendering as I had to resize the histogram for it to appear.

Reply:
We fixed that Julia, you can retest!